hugegraph-dist/src/assembly/static/bin/start-hugegraph.sh

```diff
@@ -18,12 +18,30 @@
 OPEN_MONITOR="false"
 OPEN_SECURITY_CHECK="true"
 DAEMON="true"
-PRELOAD="false"
```

Comment:
test it in binary package when we start with `bin/start-hugegraph.sh -p true` (if we remove it)

Reply:
Tested.